Makefile

```diff
@@ -15,7 +15,7 @@ ifeq ($(UNAME_S),Darwin)
 endif
 
 PB_URL="https://github.com/protocolbuffers/protobuf/releases"
-PB_VERSION=3.15.8
+PB_VERSION=3.19.4
```

Comment:
Bumping up this version because it was used in `common/segmenters/segmenters.pb.go` in https://github.com/caraml-dev/xp/pull/16